compatibility_lib/compatibility_lib/dependency_highlighter.py

```diff
@@ -73,13 +73,10 @@ def __str__(self):
                'Latest Available:\t{}\n'
                'Time Since Latest:\t{} days\n'
                '{}\n')
-        return msg.format(
-            self.name,
-            self.priority.level.name,
-            self.installed_version,
-            self.latest_version,
-            (self.current_time-self.latest_version_time).days,
-            self.priority.details)
+        return msg.format(self.name, self.priority.level.name,
+                          self.installed_version, self.latest_version,
+                          (self.current_time - self.latest_version_time).days,
+                          self.priority.details)
```

Comment:
It assigns a cost to line breaks and tries to avoid them. We need to decide whether this tool makes things better overall. If it doesn't then I should just revert this PR.

Reply:
Overall, I think yapf does a better job than flake, so I'm on board for using this tool.

I just question some of the `knobs`(?) used in the current configuration. I was skimming some of the [documentation](https://github.com/google/yapf#knobs) and am thinking that maybe by turning some of these `knobs` on/off, we can get more consistency/better readability: COALESCE_BRACKETS, DEDENT_CLOSING_BRACKETS, SPLIT_ALL_COMMA_SEPARATED_VALUES